http-body-util/src/buffered.rs

```diff
@@ -0,0 +1,112 @@
+use bytes::{Buf, Bytes};
```

Comment:
I think this file can be removed, right?

Reply:
Oh yes, sorry getting used to neovim I know how to rename a file and write it but i guess not delete the old one xD